### ISSUE TITLE
[drag&drop][table]Make Raw Records Ordering drag and drop able with DESC & ASC

### ISSUE THREAD

**Issue by junlincc** [apache/superset#16772]
https://user-images.githubusercontent.com/67837651/134255141-c4b1e0d7-b0c1-4f42-9789-b39613158239.mov

In table chart, we allow user to sort data by selecting a specific column DESC or ASC. In order to do so, users have to open a dropdown that lists ALL columns twice, with one in Desc and one in ASC. This is happening also when drag and drop is enabled, which is pretty counter intuitive, also very bad user experience! Let's make this field D&D with additional setting for DESC and ASC.

**Comment by zhaoyongjie** [comment]
This feature is not only in raw table viz, but should be redesigned `sortby` control.

**Comment by rusackas** [comment]
I agree, this is gross, and an ASC/DESC switch to cut the menu in half seems entirely sensible. Moving this to an Ideas thread to see if anyone can volunteer for a PR or spur any further discussion on the topic.
